api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.8.0"
```

Comment:
why is this changing in this PR? (and below)

Reply:
CI checks tell me should change this, otherwise CI checks failed.

Reply:
Sorry, reverting changes in swagger.json seems Okay now.